backend/amd64/emit.mlp

```diff
@@ -1506,21 +1504,8 @@ let emit_instr fallthrough i =
       I.bswap (res32 i 0);
   | Lop(Ispecific(Ibswap { bitwidth = Sixtyfour })) ->
       I.bswap (res i 0)
-  | Lop(Ispecific Isqrtf) ->
-      if arg i 0 <> res i 0 then
-        I.xorpd (res i 0) (res i 0); (* avoid partial register stall *)
```

Comment:
Hmm, I was thinking this will no longer be necessary when we start using VEX encoded operations (it no longer preserves the destination's upper bits), but that's not a good reason to remove it now...added back